Source/SwiftLintFramework/Extensions/Configuration+LintableFiles.swift

```diff
@@ -33,8 +33,16 @@ extension Configuration {
         excludeByPrefix: Bool = false,
         fileManager: LintableFileManager = FileManager.default
     ) -> [String] {
-        // If path is a file and we're not forcing excludes, skip filtering with excluded/included paths
-        if path.isFile && !forceExclude { return [path] }
+        if path.isFile {
+            if forceExclude {
+                let filteredPaths = excludeByPrefix
+                    ? filterExcludedPathsByPrefix(in: [path.absolutePathStandardized()])
+                    : filterExcludedPaths(fileManager: fileManager, in: [path.absolutePathStandardized()])
+                return filteredPaths
```

Comment:
`filteredPaths` can be inlined (directly returned).

Reply:
[cba8372](https://github.com/realm/SwiftLint/pull/4609/commits/cba8372fc5ee89ea35d5a44da57ffb463745628b)